uint16 site_id
string site_name
---
bool success